### PR TITLE
Fix handling of Metric number fields in config script

### PR DESCRIPTION
**Pull request by nkuba**
The config file contains now new properties stored as numbers: `NetworkMetricsTick` and `EthereumMetricsTick`.

The library `tomlify-j0.4` we use in config generation script requires special handling of number fields to convert them from float to integer.

We already support `Port` properties and in this PR we added support of ones which key ends with `MetricsTick`.

Refs: https://github.com/keep-network/keep-core/issues/1946

### CODE REVIEW

**Comment by pdyraga** [comment]
@nkuba Linter is not happy about this change :(

File: solidity/scripts/lcl-client-config.js [hunk 1]
```diff
@@ -44,9 +44,10 @@ module.exports = async function () {
       const formattedConfigFile = tomlify.toToml(fileContent, {
         space: 2,
         replace: (key, value) => {
-          return key == "Port" ? value.toFixed(0) : false
+          // Find keys that match exactly `Port` or ends with `MetricsTick`.
+          return key.match(/(^Port|MetricsTick)$/) ? value.toFixed(0) : false;
```
Comment (pdyraga) [inline solidity/scripts/lcl-client-config.js:48]: When running the local install I am getting:
```
Updating keep-core client configs...
Using network 'local'.

failed to update keep-core client config TypeError: key.match is not a function
    at Object.replace (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/scripts/lcl-client-config.js:47:32)
    at escapeValue_ (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:223:39)
    at escapeArray (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:172:23)
    at escapeValue_ (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:241:16)
    at escapeKeyValue (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:276:18)
    at callback (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:493:20)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:316:9)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:350:13)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:412:9)
    at escapeTable (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:511:5)
```

Comment (nkuba) [reply solidity/scripts/lcl-client-config.js:48]: Would you mind posting your config file content?

Comment (pdyraga) [reply solidity/scripts/lcl-client-config.js:48]: Sure, nothing particularly scary there I think:
```
[ethereum]
  URL = "ws://127.0.0.1:8546"
  URLRPC = "http://127.0.0.1:8545"
  MiningCheckInterval = 15
  MaxGasPrice = 40000000000

[ethereum.account]
  KeyFile = "/Users/piotr/ethereum/data/keystore/UTC--2018-11-01T06-24-04.418691008Z--65ea55c1f10491038425725dc00dffeab2a1e28a"

[ethereum.ContractAddresses]
  BondedECDSAKeepFactory = "0xc0f93b172d863fEA888665BAfaDC4852A942816f"

[SanctionedApplications]
  Addresses = [
    "0x4f7C771Ab173bEc2BbE980497111866383a21172"
  ]

[Storage]
  DataDir = "/Users/piotr/keep-ecdsa-data/1"

[LibP2P]
  Port = 4921
```

Comment (nkuba) [reply solidity/scripts/lcl-client-config.js]: Fixed in 3225538

File: solidity/scripts/lcl-client-config.js [hunk 2]
```diff
@@ -44,7 +44,25 @@ module.exports = async function () {
...
-          return key == "Port" ? value.toFixed(0) : false
+          let result
+          try {
+            result =
+              // We expect the config file to contain arrays, in such case key for
+              // each entry is its' index number. We verify if the key is a string
+              // so we can run the following match check.
+              typeof key === "string" &&
+              // Find keys that match exactly `Port` or ends with `MetricsTick`.
+              key.match(/(^Port|MetricsTick)$/)
```
Comment (pdyraga) [inline solidity/scripts/lcl-client-config.js:55]: We have two other numerical properties that we also need to handle here: `MiningCheckInterval` and `MaxGasPrice`.

Comment (pdyraga) [reply solidity/scripts/lcl-client-config.js:55]: I tested with `key.match(/(^Port|MetricsTick|MiningCheckInterval|MaxGasPrice)$/)` and everything works as expected.

**Pull request merged**